README.md

```diff
@@ -90,3 +90,87 @@ $ docker build . -t <IMAGE_TAG>
 ### Format
 
 Formatting for `PrivateKey` or `Peer` follows the format in [Nekoyume Project README][../README.md].
+
+## How to run NineChronicles Standalone on AWS EC2 instance using Docker
+
+### A) On Your Local
+
+#### Pre-requisites
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+
+#### 1. Build docker image with the tag name in [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] format.
+
+```
+ex) $ docker build . -t 9c/9c-standalone --build-arg COMMIT=9c-1
+
+Usage: docker build . -t [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] --build-arg COMMIT=[<VERSION_SUFFIX>]
+```
+
+![Docker Build](https://i.imgur.com/vc6CnbR.png)
+
+#### 2. Push your docker image to your docker hub account.
+
+```
+ex) $ docker push 9c/9c-standalone:latest
+
+Usage: docker push [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<VERSION>]
+```
+
+![Docker Push](https://i.imgur.com/IRIXXjg.png)
+
+### B) On Your AWS EC2 Instance (example instance: Ubuntu Server 20.04 LTS (HVM)/t2.micro)
+
+#### Pre-requisites
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+- AWS EC2 instance: [AWS EC2 Guide](https://docs.aws.amazon.com/ec2/index.html)
+
+#### 3. Pull your docker image to your AWS EC2 instance.
+
+```
+ex) $ sudo docker pull 9c/9c-standalone:latest
```

Comment:
IMO not good at mention `sudo` in this documents. It is depending user's system environment.

Reply:
@riemannulus  Ah ok, I did put "Ubuntu Server 20.04 LTS (HVM)/t2.micro" for the example instance but I understand what you mean. I'll delete the "sudo" commands. Also, would "ex)" be unnecessary in these example commands?

Reply:
LGTM about other command 👍